content/en/faqitems/eks-1-how-to-do-fault-injection.markdown

```diff
@@ -0,0 +1,8 @@
++++
+question = "How do I perform fault injection with Bottlerocket?"
+group = "EKS"
++++
+
+Fault injection is possible with Bottlerocket on EKS through using fault injection tools such as those in [Istio](https://istio.io/latest/docs/tasks/traffic-management/fault-injection/) or [Chaos Mesh](https://chaos-mesh.org/docs/simulate-pod-chaos-on-kubernetes/).
```

Comment:
Humm. This doesn't seem unique to Bottlerocket. Seems to me:
1) We need to talk about FI on bottlerocket specific features (unlikely?)
or
2) We need to say that Bottlerocket isn't special in this regard
or
3) Avoid this question because it's an _EKS_ question not a Bottlerocket question.

Reply:
I'm thinking (2). It might help users to be reassured that _both_ open and AWS-specific fault injection tools work with Bottlerocket. 

Maybe something like

"A Bottlerocket cluster can be tested for faults similar to how a cluster using another operating system would be tested: using fault injection tools like X, Y... or also AWS FIS"

What do you think?

Reply:
Thanks, sounds good. I'll fine-tune the sentence wording.

Reply:
How about the following?

"Fault Injection testing on a Bottlerocket cluster is not different from testing on a cluster running another operating system. You can use fault injection tools such as those in Istio or Chaos Mesh. 

Alternatively, if you are using EKS, you can use AWS Fault Injection Simulator. In order to use AWS FIS, you need SSM access to the node, which is available through the control container."

Reply:
:+1: